test/Hello_world.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
this file is not in the right place: it is a contract file, not a test file.
- it does not inherit from `Test` like `CounterTest` (see https://github.com/Amenokal-Labs/decentralized-exchange/pull/3/files#diff-8c7a37ff0156acc4a48ceac17fb6d33836dab5834df4064c2a7e5d08c8650662R7)
- extension of tests is `.t.sol`, and not `.sol`